certified-connectors/U.S. Bank Treasury Management/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2778 @@
+              "example": 5
+            },
+            "totalDebits": {
+              "type": "number",
+              "description": "Sum of all previous day detail debits(BAI record type 16's)",
+              "example": 16840.55
+            },
+            "numberOfDebits": {
+              "type": "number",
+              "description": "Item count of all previous day debits",
+              "example": 5
+            },
+            "accessibleBalance": {
+              "type": "number",
+              "description": "Available balance plus any related Zero balance account balances plus overdraft credit amount. This balance a static balance as of the time of previous day processing, and is only available if the account is entitled.",
+              "example": 271840.55
+            },
+            "balanceAsOfDate": {
+              "type": "string",
+              "description": "ISO 8601 format",
+              "example": "2020-05-03T00:00:00.000Z"
+            }
+          }
+        }
+      ]
+    },
+    "Checks": {
+      "type": "object",
+      "description": "The avilability of the fields in the below object is dependent on type of transaction.",
+      "properties": {
+        "captureDate": {
+          "type": "string",
+          "format": "date",
+          "description": "Capture date of the Check"
+        }
+      }
+    },
+    "lastRefreshedDateTime": {
+      "type": "string",
+      "description": "ISO 8601 Date format that the data is from.",
+      "example": "2020-05-03T00:00:00.000Z"
+    },
+    "ACH": {
+      "type": "object",
+      "description": "The avilability of the fields in the below object is dependent on type of transaction.",
+      "properties": {
+        "companyName": {
+          "type": "string",
+          "description": "Company Name",
+          "example": "NAME000003279"
+        },
+        "individualID": {
+          "type": "string",
+          "description": "Individual ID",
+          "example": "ACH CCD CREDIT"
+        },
+        "companyID": {
+          "type": "string",
+          "description": "Company ID",
+          "example": 1918144465
+        },
+        "secCode": {
+          "type": "string",
+          "description": "SEC Code",
+          "example": "CCD"
+        },
+        "companyDiscretionaryData": {
+          "type": "string",
+          "description": "COMPANY DISCRETIONARY DATA",
+          "example": "G"
+        },
+        "entryDescription": {
+          "type": "string",
+          "description": "Description of the entry",
+          "example": "TAXPAYMENT"
+        },
+        "effectiveEntryDate": {
+          "type": "string",
+          "format": "date",
+          "description": "Effective entry date of the transaction",
+          "example": 200130
+        }
+      }
+    },
+    "RTP": {
+      "type": "object",
+      "description": "The avilability of the fields in the below object is dependent on type of transaction.",

```

Comment:
@jzworkman all files should be free of grammatical errors and typos. in the description you have "avilability", please change to "availability".